### PR TITLE
Updates SpacemanDMM to 1.4

### PR DESCRIPTION
**Pull request by yoyobatty**
Yep.

### CODE REVIEW

**Comment by yoyobatty** [comment]
Gotta update those SHOULD_CALL_PARENT's

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
 	"eslint.workingDirectories": [
-        "./tgui-next"
+        "./tgui"
```
Comment (Neotwr) [inline .vscode/settings.json:3]: we still have tgui-next file

Comment (yoyobatty) [reply .vscode/settings.json:3]: Apparently tgui used it for it ESLint but I guess they got rid of tgui entirely

**Comment by Neotwr** [comment]
can you update this branch then I can merge

**Comment by Neotwr** [comment]
bad yoyo

**Review by Neotwr — APPROVED** [review]

**Pull request merged**